zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/archiver/ElasticsearchRepository.java

```diff
@@ -125,6 +125,7 @@ public CompletableFuture<Void> setIndexLifeCycle(final String destinationIndexNa
             .settings(
                 settings ->
                     settings.lifecycle(lifecycle -> lifecycle.name(retention.getPolicyName())))
+            .index(destinationIndexName)
```

Comment:
❓ we apply the retention period each time that we archive the indexes, so it is not expected that if we change the configuration for the retention period on the exporter configuration to affect already existing archives correct?

Reply:
Yes, it is, but there is a separate issue to tackle this :+1: